src/drivers/db/Queue.php

```diff
@@ -74,6 +74,7 @@ public function init()
      */
     public function run($repeat, $delay = 0)
     {
+        $delay = intval($delay);
```

Comment:
IMHO it is wrong way. `$delay` param comes from console command as timeout option:
https://github.com/yiisoft/yii2-queue/blob/a97007c1d76343f12f58101da8864f05e2e6f155/src/drivers/db/Command.php#L66-L69

I think, it should be made as timeout param validation:

```php
    public function actionListen($timeout = 3)
    {
        if (!is_numeric($timeout)) {
            throw new \yii\console\Exception('Timeout must be numeric.');
        }
        if ($timeout < 1) {
            throw new \yii\console\Exception('Timeout must be greater that zero.');
        }

        return $this->queue->run(true, $timeout);
    }
```